thieme-german.csl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>thieme-german.csl</title>
+    <id>http://www.zotero.org/styles/thieme-german</id>
+    <link href="http://www.zotero.org/styles/thieme-german" rel="self"/>
+    <link href="http://www.thieme.de/de/autorenlounge/fuer-zeitschriftenautoren-1789.htm" rel="documentation"/>
```

Comment:
Hi RMZelle,
congratulations to the Baby! 
Regarding your questions:
- yes, the style ist supposed to follow the Format described in "Richtig zitieren", which underwent a minor update. The new version ist attached.
[Richtig_Zitieren_20170612.pdf](https://github.com/citation-style-language/styles/files/1118722/Richtig_Zitieren_20170612.pdf)
- the Name of the new style should be "Thieme-German", since this is how we announced it already in our authors' Guidelines.
- Yes, most of These journals publish in German, and I thought we had changed the language appropriately. So if you could Change the default language to German, that would be great!